modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/MiniPlayer.kt

```diff
@@ -60,8 +62,17 @@ class MiniPlayer @JvmOverloads constructor(context: Context, attrs: AttributeSet
 
     override fun onLayout(changed: Boolean, left: Int, top: Int, right: Int, bottom: Int) {
         super.onLayout(changed, left, top, right, bottom)
-        // set the initial play button icon to the play triangle. This is run after the button has inflated or the Lottie drawable is null and it won't work.
```

Comment:
This comment is no longer valid since we have restored state and don't have to start with play button. But I'm open to suggestion if you want to keep this comment in any form.